roles/addons/prometheus_server/tasks/main.yml

```diff
@@ -6,8 +6,31 @@
     state: present
   with_items:
     - prometheus
+    - grafana
     - alertmanager
     - ipmi_exporter
+    - freeipmi
+    - snmp_exporter
+
```

Comment:
Can you also add Karma ?

Reply:
What extra repo do we need to install karma?

Reply:
Karma is autonomous. We will just need to add it as a package like prometheus and the others, in the bluebanquise infrastructure. We only package official binary, and add a service file with it.
This is not very conventional, but these tools are not that easy to build, and since these static binaries are provided by very big teams, I think we can use binaries provided on their github, to also ease support in case of issue.

Reply:
Should we open an issue in bluebanquise-infrastructure for tracking?

Reply:
Good idea. Let me do it.